easybuild/easyblocks/b/boost.py

```diff
@@ -215,3 +215,10 @@ def sanity_check_step(self):
             custom_paths["files"].append('lib/libboost_python.%s' % shlib_ext)
 
         super(EB_Boost, self).sanity_check_step(custom_paths=custom_paths)
+
+    def make_module_extra(self):
+        """Set up a BOOST_ROOT environment variable to e.g. ease Boost handling by cmake"""
+        txt = super(EB_Boost, self).make_module-extra()
```

Comment:
@jhein32 you have a syntax error here, should be `make_module_extra()`, not `make_module-extra()`

when you make changes to an existing easyblock, please make sure you test it too, by (re)building a `Boost` module in this case

you can make sure that your modified easyblock is being used via `--include-easyblocks` (see http://easybuild.readthedocs.io/en/latest/Including_additional_Python_modules.html#include-easyblocks)